riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb

```diff
@@ -0,0 +1,413 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "yh5jzMGT9Dnh"
+   },
+   "source": [
+    "***We recommend working with this notebook on Google colab***\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ridatadiscoverycenter/riddc-jbook/blob/feature-volume-converter/riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb)\n",
+    "\n",
+    "# Visualize OSOM data using the 3D volume viewer\n",
+    "\n",
+    "\n",
+    "<img src=\"https://github.com/ridatadiscoverycenter/riddc-jbook/raw/feature-volume-converter/riddc/notebooks/fox-kemper/images/volume-viewer-2.png\" >\n",
+    "\n",
+    "The following example goes through how to export Osom datasets and visualize them in 3D using the volume viewer application.\n",
+    "**NOTE: These scripts create large amount of data. We recommend to use your google drive to save them.**\n",
+    "\n",
+    "You can use the volume viewer either from you computer or using Brown's Oscar HPC.\n",
+    "\n",
+    "## 1 Get the Volume Viewer Application\n",
+    "\n",
```

Comment:
```suggestion
    "\n",
    "\n",
```

Reply:
I think this will work, just adding another space under the heading to make the section stand out more